api/src/main/java/com/codeforcommunity/dto/team/InviteContact.java

```diff
@@ -35,10 +35,11 @@ public void setEmail(String email) {
   public List<String> validateFields(String fieldPrefix) throws HandledException {
     String fieldName = fieldPrefix + "invite_contact";
     List<String> fields = new ArrayList<>();
-    if (name == null) {
+    // TODO: do we want to keep this field? It doesn't look like it's being used anywhere
```

Comment:
We're having some issues with the emailer but eventually this field will be used in the emailer.